opus/application/apps/ui/templates/ui/base.html

```diff
@@ -453,7 +453,7 @@ <h3 class='op-header-text'></h3>
     <div class="op-overlay"></div>
 
     {% include "ui/footer.html" %}
-    <div id="range_query_widget_helper_html">
+    <div id="op-range-qtype-helper-html">
```

Comment:
What do you think of renaming the id to "op-qtype-tooltip" instead?  I'm not clear that we need 'helper' or 'html' in that name and my thought is that this is just the tooltip for the qtype.